docs/src/manual/solutions.md

```diff
@@ -32,6 +32,29 @@ Subject to
  y[b] ≤ 1.0
 ```
 
+## Solutions summary  
+[`show_solution_summary`](@ref) can be used for checking the summary of the optimization solutions.
```

Comment:
A very minor thing, but please add a blank line between the title and the first line.

Reply:
I fixed